packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.load-balanced-fargate-service.ts

```diff
@@ -376,6 +376,8 @@ export = {
         hostedZoneArn: 'arn:aws:route53:::hostedzone/fakeId',
         stack,
         node: stack.node,
+        account: stack.account,
```

Comment:
Shouldn’t this be the default behavior? Why do we need to explicitly pass this in?

Reply:
This is because this test creates an "inline" implementation of `IHostedZone`, which extends `IResource`, to which I just added `account` and `region` properties.

In my opinion, this is a bad way to write tests (it should use `HostedZone`'s ``fromHostedZoneId` or `fromHostedZoneAttributes` methods instead), but I'd rather not do a big refactoring on them in this PR.

Reply:
Actually, I fixed them anyway.